src/components/results/doctorSpecialtiesListItem.js

```diff
@@ -1,14 +1,10 @@
 import React from 'react';
 
-class DoctorSpecialtiesListItem extends React.Component {
-  
-    render() {
-        console.log(this.props.specialties);
-      return (
-      <li className="specialties-list__item">{this.props.specialties}</li>
-      )
-    }
-  }
+const DoctorSpecialtiesListItem = ({specialty}) => {
```

Comment:
[comment/possible fix] There's nothing specifically wrong with this, but I would say this is probably an example of not creating a specific component for every single thing. In this case, all you're doing is rendering an item from a list, and there probably won't be much reuse you can get out of this, so it'd make sense to just have it in the list component instead.